apps/wallet/src/ui/app/hooks/useDeleteAccountSourceMutation.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useMutation } from '@tanstack/react-query';
+import { useBackgroundClient } from './useBackgroundClient';
+
+export type DeleteType = 'mnemonic';
+
+export function useDeleteAccountSourceMutation() {
+	const backgroundClient = useBackgroundClient();
+	return useMutation({
+		mutationKey: ['delete account source'],
+		mutationFn: async ({ type }: { type: DeleteType }) => {
+			return await backgroundClient.deleteAccountSourceByType({ type });
```

Comment:
it seems we are not passing the entropy provided by the user and we always create a new one